audio/Makefile

```diff
@@ -38,10 +38,11 @@ LINUXHEADERSDIR := ../third_party/fx2lib-linux-headers
 COMMON_DIR := ../common
 INCS := -I$(LINUXHEADERSDIR)
 
-CFLAGS := -DDSCR_AREA=0x3e00 --code-size 0x3c00 --xram-size 0x0200
+# Descriptors require more space in memory
```

Comment:
Maybe? `audio descriptors require more space, so using 3d00 rather than the normal 0x3c00`

Reply:
Fixed